src/test/groovy/com/getkeepsafe/dexcount/DexFileTest.groovy

```diff
@@ -0,0 +1,22 @@
+package com.getkeepsafe.dexcount;
+
+import spock.lang.Specification
+
+class DexFileTest extends Specification {
+    def "test AAR dexcount"() {
+        setup:
+        def currentDir = new File(".").getAbsolutePath()
+
+        when:
+        def aarFile = new File(currentDir + File.separatorChar + "src"
+                + File.separatorChar + "test" + File.separatorChar + "resources"
+                + File.separatorChar + "android-beacon-library-2.7.aar")
```

Comment:
The other option might be `resources/android-beacon-library-2.7.aar`.  Java is such a pain sometimes...